src/cli_pretty_printing/mod.rs

```diff
@@ -47,3 +82,41 @@ macro_rules! human_checker_check {
         )
     };
 }
+
+/// When Ares has failed to decode something, print this message
+pub fn failed_to_decode() {
+    let config = crate::config::get_config();
+    if !config.api_mode {
+        // The program can roughly do 45 decodings a second
+        // Currently it is not possible to get this info at this stage of the program from the decoding level
+        // TODO fix this
+        let ares_decodings = config.timeout * 45;
+        let time_took = calculate_time_took(ares_decodings);
+        println!("Ares has failed to decode the text after trying {} decodings. If you would have used Ciphey, it would have taken you {}", ares_decodings, time_took);
+        println!("If you want more help, please ask in #coded-messages in our Discord http://discord.skerritt.blog")
+    }
+}
+/// Calculate how long it would take to decode this in Ciphey
+pub fn calculate_time_took(decoded_times_int: u32) -> String {
+    // TODO if we grab how long the programs been running for (see timer) we can make some nice stats like:
```

Comment:
Instead of calculating how many decodings per second Ares did based off the timer, why don't we output that like how we did with the depth level?

Reply:
I am not sure what you mean?

Reply:
I'm talking about the TODO:
> // TODO if we grab how long the programs been running for (see timer) we can make some nice stats like:

We shouldn't do this and instead Ares should output the actual number of decodings it did. Not based off the timer.